archeo/style.css

```diff
@@ -198,8 +199,19 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 }
 
 /*
- * We need this until https://github.com/WordPress/gutenberg/issues/38498 is fixed.
+ * Needed until https://github.com/WordPress/gutenberg/issues/38498 is fixed.
  */
 .wp-block-group.simple-list-of-posts {
 	align-items: baseline;
 }
+
+/*
+ * Needed until https://github.com/WordPress/gutenberg/pull/38986 is merged.
+ */
+.wp-block-group.simple-list-of-posts a {
```

Comment:
Do we want to tie this to the custom class name, or should it apply to all instances of the block?

Reply:
Hmm, I guess I was thinking of this as being a specific pattern style, but maybe we do want this to apply to all post title blocks. Now you've mentioned it, I'm thinking this would probably be the better default style for users.

@kjellr @beafialho, what do you think?

Reply:
Making it the default for all Post Titles sounds good to me.

Reply:
Thanks @kjellr, I've applied these styles to all Post Titles now.